_automating-configurations/workflow-templates.md

```diff
@@ -6,9 +6,11 @@ nav_order: 25
 
 # Workflow templates
 
-OpenSearch provides several workflow templates for some common machine learning (ML) use cases, such as semantic or conversational search. 
+OpenSearch provides several workflow templates for some common machine learning (ML) use cases. Using a template simplifies complex setups required for use cases like semantic or conversational search. 
```

Comment:
Can we explain here maybe that we also support pre defined template based on a given use case? and also user can give there own template if they want and then link to create workflow API?
Then we can give example: